app/src/main/res/values-hi-rIN/strings.xml

```diff
@@ -0,0 +1,145 @@
+    <string name="msg_avatar_url">अवतार यूआरएल</string>
+    <string name="msg_or_continue_using_social_accounts">या सामाजिक खाते का उपयोग करना जारी रखें</string>
+    <string name="msg_new_user">नया उपयोगकर्ता? %1$s</string>
+    <string name="msg_new_user_agreement">आगे बढ़कर आप हमारे %1$s और %2$s से सहमत हो रहे हैं</string>
+    <string name="msg_2fa_code">कोड 2FA</string>
+    <string name="msg_more_than_ninety_nine_unread_messages" translatable="false">99+</string>
+    <string name="msg_yesterday">कल</string>
+    <string name="msg_message">संदेश</string>
+    <string name="msg_this_room_is_read_only">यह रूम केवल पढ़ने के लिए है</string>
+    <string name="msg_invalid_2fa_code">अमान्य 2FA कोड</string>
+    <string name="msg_invalid_file">अवैध फाइल</string>
+    <string name="msg_invalid_server_url">अमान्य सर्वर यूआरL</string>
+    <string name="msg_content_description_log_in_using_facebook">Facebook द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_github">Github द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_google">Google द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_linkedin">Linkedin द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_meteor">Meteor द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_twitter">Twitter द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_gitlab">Gitlab द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_send_message">मेसेज भेजें</string>
+    <string name="msg_content_description_show_attachment_options">अटैचमेंट विकल्प दिखाएं</string>
+    <string name="msg_you">आप</string>
+    <string name="msg_unknown">अनजान</string>
+    <string name="msg_email_address">ईमेल पता</string>
+    <string name="msg_utc_offset">यूटीसी ऑफ़सेट</string>
+    <string name="msg_new_password">नया पासवर्ड दर्ज करें</string>
+    <string name="msg_confirm_password">नए पासवर्ड की पुष्टि करें</string>
+    <string name="msg_unread_messages">अपठित संदेश</string>
+
+    <!-- System messages -->
+    <string name="message_room_name_changed">%2$s ने रूम का नाम बदलकर %1$s किया</string>
+    <string name="message_user_added_by">उपयोगकर्ता %1$s द्वारा %2$s को जोड़ा गया</string>
+    <string name="message_user_removed_by">उपयोगकर्ता %1$s द्वारा %2$s को निकाला गया</string>
+    <string name="message_user_left">ने चैनल को छोड़ दिया है</string>
+    <string name="message_user_joined_channel">चैनल में शामिल हो गया है</string>
+    <string name="message_welcome">%s का स्वागत करते हैं</string>
+    <string name="message_removed">संदेश हटाया गया</string>
+    <string name="message_pinned">एक संदेश पिन किया:</string>
+
+    <!-- Message actions -->
+    <string name="action_msg_reply">जवाब दें</string>
+    <string name="action_msg_edit">संपादन करें</string>
+    <string name="action_msg_copy">प्रतिलिपि</string>
```

Comment:
May be "नकल" or "कापी" will be better

Reply:
I think "कॉपी" will be better than "नकल"